lib/config/config_test.go

```diff
@@ -430,6 +431,76 @@ func TestFolderPath(t *testing.T) {
 	}
 }
 
+func TestFolderCheckPath(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("windows")
```

Comment:
The test should run on Windows, this already worked afaik on other platforms. The build server runs in developer mode and we should be able to create symlinks just fine, so this test should pass. Not everywhere is the same of course which is why I suggest a t.Skip on error from os.Symlink

Reply:
... in theory. I tested it now, and while I can mklink just fine from a command prompt without any elevation I still get `symlink foo ~windows-symlink-test-8366981916958631613: A required privilege is not held by the client.` from tests which is annoying

Reply:
Ah. Not only must symlinks be enabled for normal users, the *application* must also say "please let me create a symlink even though I'm just a normal process", using the SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE flag in the symlink call. That's so fucking stupid.

https://msdn.microsoft.com/en-us/library/windows/desktop/aa363866(v=vs.85).aspx

Reply:
So I guess this is fine, once working, I filed the other one on Go. Maybe you want to contribute the fix there. :)